### PR TITLE
DPC-1441 Feature: Proxy GET request to attribution service for Implementer Org

### PR DESCRIPTION
**Pull request by charmcitygavin**
### Fixes [DPC-1441](https://jira.cms.gov/browse/DPC-1441)

Proxies a GET request to the attribution service and gets a list of organizations associated for an implementer.

### Proposed Changes

* Proxies the GET request to the attribution service
* Returns a JSON-formatted list of all orgs associated with the specified implementer

### Change Details

NOTE: This is built off of existing work in [DPC-1440](https://jira.cms.gov/browse/DPC-1440). It would be a good idea to wait until that PR is merged into `master` before attempting to merge this one.

### Security Implications

- [ ] new software dependencies

- [ ] security controls or supporting software altered

- [ ] new data stored or transmitted

- [ ] security checklist is completed for this change

- [ ] requires more information or team discussion to evaluate security implications

- [x] no PHI/PII is affected by this change

### Acceptance Validation

I wrote an initial test, but some additional tests couldn't hurt. I also manually tested the endpoint with Postman.

Screen Shot 2021-07-08 at 2 57 01 PM

### Feedback Requested

Any and all feedback welcome, as always.

### CODE REVIEW

File: dpc-go/dpc-api/src/client/attribution_client.go [hunk 1]
```diff
@@ -106,6 +107,54 @@ func (ac *AttributionClient) CreateImplOrg(ctx context.Context, body []byte) ([]
 	return b, nil
 }
 
+// GetImplOrg calls attribution service via GET to return the Organizations associated with an Implementer
+func (ac *AttributionClient) GetImplOrg(ctx context.Context, resourceType ResourceType) ([]byte, error) {
```
Comment (MrBilnon) [inline dpc-go/dpc-api/src/client/attribution_client.go:111]: resourceType probably should just be hardcoded as it doesn't need to change, can always be `implementer

File: dpc-go/dpc-api/src/client/attribution_client.go [hunk 2]
```diff
@@ -106,6 +107,54 @@ func (ac *AttributionClient) CreateImplOrg(ctx context.Context, body []byte) ([]
...
+	}
+	resp, err := ac.httpClient.Do(req)
+	if err != nil {
+		log.Error("Failed to send request", zap.Error(err))
+		return nil, errors.Errorf("Failed to send request")
+	}
+
+	if resp.StatusCode < 200 || resp.StatusCode > 299 {
+		log.Error(fmt.Sprintf("Failed to send request. Status code %d", resp.StatusCode))
+		return nil, errors.Errorf("Failed to retrieve resource")
```
Comment (MrBilnon) [inline dpc-go/dpc-api/src/client/attribution_client.go:140]: All the error messages should probably be related to the implementer/id/org specific call

File: dpc-go/dpc-api/src/client/attribution_client.go (hunk 1 above)
Comment (dhgreene) [reply dpc-go/dpc-api/src/client/attribution_client.go:111]: Looks like we depend on `ResourceType` being the second parameter here, but I've added a check to ensure it's always `ResourceType(Implementer)`.

Comment (MrBilnon) [reply dpc-go/dpc-api/src/client/attribution_client.go:111]: yea you will never have any other resource type for this function so might as well just hard code it

Comment (dhgreene) [reply dpc-go/dpc-api/src/client/attribution_client.go]: OK, done.

**Review by MrBilnon — APPROVED** [review]

**Pull request merged**